README.md

```diff
@@ -15,6 +15,20 @@ docker pull k8s.gcr.io/scheduler-plugins/controller:$TAG
 
 You can find [how to install release image](doc/install.md) here.
 
+## Plugins
+
+The kube-scheduler binary includes the below list of plugins. They can be configured by creating one or more
+[scheduler profiles](https://kubernetes.io/docs/reference/scheduling/config/#multiple-profiles).
+
+* [Capacity Scheduling](pkg/capacityscheduling/README.md)
+* [Coscheduling](pkg/coscheduling/README.md)
+* [Cross Node Preemption](pkg/crossnodepreemption/README.md)
```

Comment:
Let's put the sample plugins in another stanza. (Cross Node Preemption, Pod State and Quality of Service).

Reply:
@Huang-Wei fixed in commit 684473aa4512f8597c5b986ed30da1d09e8cc649. I can squash commits before merging. Thanks!